src/main/java/com/alkemy/ong/controller/UserController.java

```diff
@@ -0,0 +1,47 @@
+package com.alkemy.ong.controller;
+
+import com.alkemy.ong.domain.User;
+import com.alkemy.ong.dto.UserCreationDTO;
+import com.alkemy.ong.mapper.UserMapper;
+import com.alkemy.ong.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.http.HttpStatus;
+
+import javax.validation.Valid;
+import java.util.HashMap;
+import java.util.Map;
+
+@RestController
+public class UserController {
+
+    @Autowired
```

Comment:
Se sugiere no usar `@Autowired` en Spring, y  en vez de eso usar inyección de dependencia a traves de los constructores. Ejemplo:
```suggestion
    private final UserService userService;

    public UserController(UserService userService) {
        this.userService = userService;
    }
```